src/app/utils/structure-field-properties/relation-property/relation-property.component.ts

```diff
@@ -335,6 +335,11 @@ export class RelationPropertyComponent implements OnInit, OnDestroy, AfterViewIn
         return;
       }
     }
+    for(var i=0;i<list.value?.length;i++){
```

Comment:
Change this code to not use a for loop, for use list.value.<array_method>

Reply:
change is done, thanks